tests/test-device-manager.cpp

```diff
@@ -75,10 +75,11 @@ class MainDeviceManager
         , _frame_height (0)
         , _frame_count (0)
         , _frame_save (0)
-        , _enable_display (false)
+        , _need_display (false)
     {
 #if HAVE_LIBDRM
-        _display = DrmDisplay::instance();
+        _display = DrmDisplay::instance ();
+        _need_display = _display->get_preview ();
```

Comment:
how about keep _enable_display and add a new member _can_display. 
e.g.

    _can_display = _display->can_preview ();